modules/http/src/main/java/org/mule/module/http/internal/request/HttpClient.java

```diff
@@ -30,6 +31,6 @@
     /**
      * Sends a HttpRequest without blocking the current thread.  When a response is available or the request times out the provided CompletionHandler will be invoked.
      */
-    public void send(HttpRequest request, int responseTimeout, boolean followRedirects, HttpRequestAuthentication authentication, final CompletionHandler<HttpResponse, Exception> handler);
+    public void send(HttpRequest request, int responseTimeout, boolean followRedirects, HttpRequestAuthentication authentication, final CompletionHandler<HttpResponse, Exception> handler, WorkManager workManager);
```

Comment:
Can you set a more meaningful name for the workManager?. Something like responseProcessingWorkManager? or Javadoc ?